CRM/Contact/Import/Parser/Contact.php

```diff
@@ -3641,4 +3638,29 @@ protected function getRelationshipLabel(int $id, string $direction): string {
     return $this->relationshipLabels[$id . $direction];
   }
 
+  /**
+   * Transform the input parameters into the
```

Comment:
This comment kinda trails off into the...

Reply:
ug - given the conflict issues we've been having maybe I'll fix in a follow up